server/lib/providers/bitbucketApi.js

```diff
@@ -20,42 +20,10 @@ export default class Bitbucket {
       }
     }, options);
 
-    this.request = (opts, cb) => {
-      request(opts, (error, response, data) => {
-        if (error) {
-          cb(error);
-        } else {
-          try {
-            data = JSON.parse(data); // eslint-disable-line no-param-reassign
-            if (typeof data.errors !== 'undefined') {
-              cb(data.errors);
-            } else if (response.statusCode !== 200) {
-              cb(this.generateApiError(opts.url, response.statusCode, response));
-            } else {
-              cb(null, data);
-            }
-          } catch (parseError) {
-            if (response.statusCode === 200) {
-              cb(null, data);
-            } else {
-              cb(this.generateApiError(opts.url, response.statusCode, response));
-            }
-          }
-        }
-      }).auth(this.options.user_name, this.options.password, true);
-    };
-
-    this.generateApiError = (url, statusCode, response = null) => {
```

Comment:
What are we losing by removing this?

Reply:
`axios` returns request errors as `error` opposing to `request`, which tend to return errors as `response.errors`, while `error` is null. with `axios` we don't need to decorate those errors